src/environment.py

```diff
@@ -58,16 +43,16 @@ def _env_for_clustering(config: dict, app_name: str) -> dict:
         dict of clustering settings, varying the cluster name on the application name.
     """
     if not config.get("clustering"):
-        return {}
-    return {
-        "MM_CLUSTERSETTINGS_ENABLE": "true",
+        return ()
+    return (
+        ("MM_CLUSTERSETTINGS_ENABLE", "true"),
```

Comment:
Another option would be to put a `yield` in front of each of these lines and replace the `return ()` with just `return` - then the function is a generator

Reply:
Changed to generator